python/graphscope/framework/dag_utils.py

```diff
@@ -659,17 +659,12 @@ def unload_graph(graph):
     """Unload a graph.
 
     Args:
-        graph (:class:`Graph`): The graph to unload.
+        graph (:class:`GraphDAGNode`): The graph to unload.
 
     Returns:
         An op to unload the `graph`.
     """
     config = {}
-    if not isinstance(graph, DAGNode):
-        config.update({types_pb2.GRAPH_NAME: utils.s_to_attr(graph.key)})
-        # Dynamic graph doesn't have a vineyard id
-        if hasattr(graph, "vineyard_id"):
-            config[types_pb2.VINEYARD_ID] = utils.i_to_attr(graph.vineyard_id)
```

Comment:
The `unload` doesn't require the `vineyard_id` anymore?

Reply:
move this logic to the coordinator.